TrafficSimulator/Assets/CarSpawner.cs

```diff
@@ -0,0 +1,254 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Vehicle = DataModel.Vehicle;
+using AutoDrive = Car.AutoDrive;
+
+namespace RoadGenerator
+{
+    public class CarSpawner : MonoBehaviour
+    {
+        [SerializeField] private GameObject carPrefab;
+        [SerializeField] private GameObject roadSystem;
```

Comment:
Coding standard

Reply:
Should change the name to roadSystemObject as there is another variable called _roadSystem